sequencer/src/persistence/sql.rs

```diff
@@ -1824,23 +1824,23 @@ impl SequencerPersistence for Persistence {
         tx.commit().await
     }
 
-    async fn load_state_cert(&self) -> anyhow::Result<LightClientStateUpdateCertificate<SeqTypes>> {
-        match self
+    async fn load_state_cert(
+        &self,
+    ) -> anyhow::Result<Option<LightClientStateUpdateCertificate<SeqTypes>>> {
+        let Some(row) = self
             .db
             .read()
             .await?
-            .fetch_one("SELECT state_cert, MAX(epoch) from state_cert GROUP BY epoch")
-            .await
-        {
-            Ok(row) => {
-                if let Some(data) = row.get::<Option<Vec<u8>>, _>("state_cert") {
-                    bincode::deserialize(&data)
-                        .context("deserializing light client state update certificate")
-                } else {
-                    Ok(LightClientStateUpdateCertificate::genesis())
-                }
-            },
-            Err(_) => Ok(LightClientStateUpdateCertificate::genesis()),
+            .fetch_optional("SELECT state_cert from state_cert ORDER BY epoch DESC LIMIT 1")
+            .await?
+        else {
+            return Ok(None);
+        };
+        if let Some(data) = row.get::<Option<Vec<u8>>, _>("state_cert") {
```

Comment:
why is this deserialized as Option?

Reply:
check [5ccfe57](https://github.com/EspressoSystems/espresso-network/pull/2762/commits/5ccfe570049d0bd9533944dbb5de361864762787)

Also add some new test cases